prow/config.yaml

```diff
@@ -14278,3 +14278,40 @@ periodics:
         - name: cache-ssd
           hostPath:
             path: /mnt/disks/ssd0
+
+- name: periodic-test-infra-retester
+  interval: 20m  # Retest at most 1 PR per 20m, which should not DOS the queue.
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/commenter:latest
+      args:
+      - |-
+        --query=is:pr
+        -label:do-not-merge
+        label:lgtm
+        label:approved
+        -status:pending
+        -status:success
+        -label:needs-rebase
+        -label:needs-ok-to-test
+        -label:"cncf-cla: no"
+        repo:kubernetes/kubernetes
+      - --user=fejta-bot
+      - --token=/etc/token/bot-github-token
+      - |-
+        --comment=/retest
+        This bot automatically retries jobs that failed/flaked on approved PRs (send feedback to `@fejta`).
+
+        Review the [full test history](https://k8s-gubernator.appspot.com/pr/{{.Number}}) for this PR.
+
+      - --template
+      - --ceiling=1
+      - --wet
+      volumeMounts:
+      - name: token
+        mountPath: /etc/token
+    volumes:
+    - name: token
+      secret:
+        secretName: fejta-bot-token
```

Comment:
The `-label:needs-ok-to-test` should filter out PRs that the bot cannot retest. 

However there are actually some PRs that were validated with an `@k8s-bot ok to test` command awhile ago, which we no longer recognize. 

When I detect these I've been going in and adding an `/ok-to-test` comment for these PRs.